website/shared/markdown/time_test.go

```diff
@@ -0,0 +1,40 @@
+package markdown
+
+import (
+	"bytes"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/yuin/goldmark"
+)
+
+type timecase struct {
+	name, in, out string
+}
+
+func TestTimeExtension(t *testing.T) {
+	cases := []timecase{
+		{"valid", "{26 Dec 25 10:30 +0000}", `<p><time datetime="1766745000" data-dur="0" data-type="local">26 Dec 25 10:30 +0000</time></p>`},
+		{"valid with dur", "{26 Dec 25 10:30 +0000|1h}", `<p><time datetime="1766745000" data-dur="3600000" data-type="local">26 Dec 25 10:30 +0000</time></p>`},
+		{"invalid", "{26 Dec 25 1as2d|1h}", `<p>{26 Dec 25 1as2d|1h}</p>`},
+		{"opening with no time", "{: this is a weird smily {:", `<p>{: this is a weird smily {:</p>`},
+		{"opening with closing no time", "this is a valid brace with nothing good in it {}", `<p>this is a valid brace with nothing good in it {}</p>`},
+		{"newlinw", "newline time {yes \n}", "<p>newline time {yes<br>\n}</p>"},
```

Comment:
If there is a valid format here instead of just "yes", does this still render?

For example, if you made it `{26 Dec 25 10:30 +0000|1h\n}`. Does this work?